tests/sentry/integrations/github/test_search.py

```diff
@@ -38,6 +40,17 @@ def _create_integration(self):
             },
         )
 
+    def _build_repo_query_path(self, *, query: str) -> str:
+        return f"{self.base_url}/search/repositories?" + urlencode(
+            {
+                "q": build_repository_query(
+                    metadata=self.integration.metadata,
+                    name=self.integration.name,
+                    query=query,
+                ).decode("utf-8")
+            }
+        )
+
```

Comment:
This test shouldn't double check the query structure, so pull in the method and just use its output. Don't want to update all these location if we wanna change the query later.